crates/ott-balancer/src/config.rs

```diff
@@ -19,6 +19,11 @@ pub enum MonolithSelectionStrategy {
     MinRooms(MinRoomsSelector),
 }
 
+impl Default for MonolithSelectionStrategy {
+    fn default() -> Self {
+        MonolithSelectionStrategy::MinRooms(MinRoomsSelector)
+    }
+}
```

Comment:
You should be able to `#[derive(Default)]` and select which enum variant is the default by tagging it with the `#[default]` attribute.

Reply:
I tried to do that, but compiler gave me the error that I can't do that since it's not a unit variant or something along the lines.

Reply:
Ah ok. Don't worry about it then.